composition/README.md

```diff
@@ -147,9 +147,12 @@ When executed correctly, strings should be printed to terminal similar to what i
 
 ## **FAQ**
 
-WIP
+`Q`: Why use Composition?
+
+`A`: Composition provides utilties to  manage life cycle for nodes. This provides greater control over the state of a given ROS-based system. The idea of a managed node is to provide a interactable interface which executes to a known life cycle state machine, allowing for higher system transparency and code-agnosticism.
```

Comment:
This section is describing lifecyle/managed nodes, not node composition. Node composition allows nodes to reside in the same process where the overhead of marshaling and unmarshaling messages can be avoided.

Reply:
Noted. Updated the section to the following:

```md
`Q`: Why use node composition?

`A`: Node composition avoids the overhead of marshaling and unmarshalling messages by allowing nodes to be instantiated within the same process.
```

Reply:
Done.